### DISCUSSION TITLE
Send from email

### DISCUSSION THREAD

**Discussion by ZobeerWebDeveloper** [emailjs-com/help#8] [Q&A] [answered]
in my template i want to send emails from another email which is dynamic variables {{userEmail}}, but emailjs still sending emails from my personal email

**Comment by xr0master** [comment, marked as answer]
Changing the "from" email address dynamically is highly not recommended - the email servers will assume you are impersonating another person. These emails will most probably go to spam folders or be blocked. Almost all personal email services ignore the "From" field and use the default one.

Instead, I would suggest putting the contact person's name in the "From Name" field and the person's email address in the "Reply To" field. The recipient will see that the email is from "Tony", and if the recipient clicks the reply button, the "To" field will be populated with the value "Reply To" (Tony's email).

If it's important to set the specific email address in the "From Email" field, I suggest you create the required address, for example, no-reply@my-company.com, and connect it as the email service.
